### ISSUE TITLE
This is not exactly a bug report, but a solution you can do...

### ISSUE THREAD

**Issue by Looneybinke903** [Genymobile/scrcpy#2133]
When I downloaded it I couldn't start because it said: "Adb push! Returned with value 1" so I looked for solutions and in one of the places I found someone who said to put "-m 1080" that would solve and be resolvable, in a way, but I still had to enter on the windows Promt, fix the directory (cd c: \ scrcpy) and put the "mallet", scrcpy -m 1080, but this was very exhausting to do every time so I remembered once to put a command in the app it was just go on property and write in the destination, give it a space and put " - " and the command that at the time you use the app would work with the proposed command so I created a shortcut on the desktop I did the following:
![image](https://user-images.githubusercontent.com/58985545/108529157-f4b24100-72b2-11eb-9793-cdf3c4a051b4.png)

> C: \ ScrCPy \ scrcpy.exe -m 1080 -f -w --turn-screen-off --no-key-repeat --prefer-text

**-m 1080:** to work

**-f:** full screen (I indicate if only the top one is on a green screen that when it enters full screen it corrects Ps: there is no need to keep it in full screen!)
![image](https://user-images.githubusercontent.com/58985545/108529602-64c0c700-72b3-11eb-9bed-bfab305e6d0c.png)

**-w:** stay awake, to prevent the device to sleep after some delay when the device is plugged in

 **--turn-screen-off:** It is possible to turn the device screen off while mirroring on start with a command-line option (Or by pressing MOD + o at any time. To turn it back on, press MOD + Shift + o )

**--prefer-text:** (when using accents repeated the letter (EX: _aã_)
There are two kinds of events generated when typing text:

* key events, signaling that a key is pressed or released;
text events, signaling that a text has been entered.
* By default, letters are injected using key events, so that the keyboard behaves as expected in games (typically for WASD keys).

But this may cause issues. If you encounter such a problem, you can avoid it by.

After applying, I could now enter srcpy in a faster and more viable way!
(to make it more "comfortable" I changed the icon of the executable to a phone.)
![image](https://user-images.githubusercontent.com/58985545/108529075-dc422680-72b2-11eb-8b22-abba45337c1d.png)

> @rom1v **So, what do you think ??**

**Comment by rom1v** [comment]
Yes, you can add arguments directly on the Windows shortcut. Alternatively, you could also add arguments in a script, as mentioned here: https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#command-line-on-windows

**Comment by Looneybinke903** [comment]
> Yes, you can add arguments directly on the Windows shortcut. Alternatively, you could also add arguments in a script, as mentioned here: https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#command-line-on-windows

Is that in my case I don't have much experience in the programming area (I'm still taking courses) and I usually try to do a low spec of games since my PC has an Inspiron 1545 Pentium (R), Dual-Core CPU T4500 @ 2.30GHz, Mobile Intel (R) 4 Series Express Chipset Family. So I used all my neural capacity to make this invasive.

How I like to tell my friends when I can do something crazy: I have transcended!
And I couldn't access that file, doing what I did

but I'm happy if you have analyzed my simpler point of view more functional
